frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -47,10 +47,9 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   const entities = useMemo(() => {
     if (!users) return []
 
-    const userKind = users.find((userGroup) => userGroup.kind === 'user')
-    if (userKind) {
-      // return userKind.entities.map((entity) => entity.split(':')[1])
-      return userKind.entities
+    const usrGroup = users.find((usrGroup) => usrGroup.kind === 'user')
```

Comment:
Although not a problem due to scoping being a thing, could we change to the following to avoid confusion between the two vars?
```
const userGroup = users.find((usrGroup) => usrGroup.kind === 'user')
```